.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Issue Type 
+
+(check one with "x")
+
+```
+[ ] bug report
+[ ] feature request
+```
+
+
+## Current Behavior
+
+
+## Expected Behavior
+
+
+## Steps to Reproduce
+
+
+## Motivation for Change
+
+
+## System Specs
+
+- Python Version:
```

Comment:
Can you add these two as well?
pyethereum Version: x.x.x
Environment (output of pip freeze):